docs/source/optimization_ov.mdx

```diff
@@ -112,6 +112,76 @@ metrics = trainer.evaluate()
 trainer.save_model()
 ```
 
+## Distillation
+Distillation is an effective method to improve model performance by learning from the knowledge of, usually, larger model. `OVTrainer` also provides the capability to perform distillation during training. Users are required to load the teacher and feed the loaded model to `OVTrainer` as follows. Subclassed from `TrainingArguments`, `OVTrainingArguments` is provided to ease users to specify distillation hyperparameters (such as teacher model path, distillation weightage and temperature), as well as to expose the hyperparameters as commandline arguments to the training script.
+```diff
```

Comment:
The documentation build shows some formatting issues. I would try to add a blank line before this. See https://moon-ci-docs.huggingface.co/docs/optimum-intel/pr_150/en/optimization_ov

Reply:
thanks for the suggestion! next iteration here https://github.com/huggingface/optimum-intel/pull/150/commits/c42936c5796db560491b913e80eecd6aef38c625